wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -1697,7 +1697,9 @@ public final class MLSService: MLSServiceInterface {
                 subconversationType: subconversationType
             )
         } catch DecryptionError.wrongEpoch {
-            await fetchAndRepairGroupIfPossible(with: groupID)
+            Task.detached { [self] in
+                await fetchAndRepairGroupIfPossible(with: groupID)
```

Comment:
please add a comment why this is detached, also wonder what caller do on this? apparently just log.
So how does it retry to decrypt it?

Reply:
added comment